ovos_tts_server/__main__.py

```
@@ -41,13 +38,8 @@ def main():
     parser.add_argument("--badge", help="URL of visitor badge", default=None)
     args = parser.parse_args()
 
-    server, engine = start_tts_server(args.engine, cache=bool(args.cache),
-                                      has_gradio=bool(args.gradio))
+    server, engine = start_tts_server(args.engine, cache=bool(args.cache))
```

Ruff (0.14.13)

Unpacked variable `engine` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)